web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -31,6 +31,7 @@ mutationAssessor.url=http://mutationassessor.org/r3/?cm=var&var=VARIANT&frm=json
 # my variant info web API URL
 #"VARIANT" will be replaced with actual variant upon each request.
 myvariantinfo.url=https://myvariant.info/v1/variant/VARIANT
+myvariantinfo.max_page_size=200
```

Comment:
200 works fine locally, not sure if it is the optimal value though

Reply:
maybe ok to comment this out? E.g. if we want to change the default later this avoids having people to update their property files

Reply:
@inodb we still want to keep the default value defined [here](https://github.com/genome-nexus/genome-nexus/pull/413/files#diff-0fbc832c37973b03f872a4fb740b8ae8R40), right?

Reply:
yup that's fine